src/Controller/UserController.php

```diff
@@ -121,19 +132,53 @@ function postDeleteAction(Application $app, $userName) {
 
     function calendarAction(Application $app, $userName) {
         $calendars = $app['sabredav.backend.caldav']->getCalendarsForUser('principals/' . $userName);
+        $calendarsData = [];
 
+        foreach ($calendars as $calendar) {
+            $calendarId = $calendar['id'];
+            $calendar['eventCount'] = count($app['sabredav.backend.caldav']->getCalendarObjects($calendarId));
```

Comment:
This is not really a good name, as it also includes tasks and journals and whatever else people might store.

So this should either be `calendarObjectCount` or create separate `eventCount` `todoCount` and `journalCount`.